modules/programs/zsh.nix

```
@@ -149,6 +149,14 @@ in
     programs.zsh = {
       enable = mkEnableOption "Z shell (Zsh)";
 
+      autocd = mkOption {
+        default = false;
+        description = ''
+        automatically enter into a directory if typed directly into shell.
```

The description should start with a capital letter, also can fix the indentation:
```suggestion
          Automatically enter into a directory if typed directly into shell.
```